PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/referencebook/AzureSdkTreePanel.java

```diff
@@ -129,6 +158,42 @@ private void loadData(final List<? extends AzureSdkServiceEntity> services, Stri
         TreeUtil.promiseSelectFirstLeaf(this.tree);
     }
 
+    private String getServiceKeyByName(final String name) {
+        return StringUtils.lowerCase(StringUtils.trim(name));
+    }
+
+    private void loadServiceData(AzureSdkServiceEntity service, AzureSdkCategoryEntity categoryService, DefaultMutableTreeNode categoryNode, String... filters) {
```

Comment:
DefaultMutableTreeNode -> TreeNode

Reply:
updated in https://github.com/microsoft/azure-tools-for-java/pull/5400